solvers/glmnet.py

```diff
@@ -24,9 +25,13 @@ class Solver(BaseSolver):
         'for generalized linear models via coordinate descent", '
         'J. Stat. Softw., vol. 33, no. 1, pp. 1-22, NIH Public Access (2010)'
     ]
-    stop_strategy = 'iteration'
     support_sparse = False
 
+    stopping_criterion = SufficientProgressCriterion(
+        patience=7, eps=1e-38, strategy='tolerance')
+    # We use the tolerance strategy because if maxit is too low and glmnet
+    # convergence check fails, it returns an empty model
```

Comment:
Nitpick: I'd rather have the comments before so we know what happen next when reading.
```suggestion
    # We use the tolerance strategy because if maxit is too low and glmnet
    # convergence check fails, it returns an empty model
    stopping_criterion = SufficientProgressCriterion(
        patience=7, eps=1e-38, strategy='tolerance'
    )
```

Reply:
yes it makes sense indeed, I will use that in the future